tardis/io/config_reader.py

```diff
@@ -806,9 +806,9 @@ class Configuration(ConfigurationNameSpace):
     """
 
     @classmethod
-    def from_yaml(cls, fname, test_parser=False):
+    def from_yaml(cls, fname, test_parser=False, loader=yaml.Loader):
         try:
-            yaml_dict = yaml.load(open(fname), Loader=YAMLLoader)
```

Comment:
Can you please rebase and introduce this commit as a fixup of a10d98d8fbf75272a602d29c8752f87c60e6dce4 ?
That prevents switching back and forth between yaml loaders.